code/game/objects/structures/plaques/static_plaques.dm

```diff
@@ -141,3 +141,8 @@
 /obj/structure/sign/plaques/kiddie/gameoflife
 	name = "\improper Conway's The Game Of Life plaque"
 	desc = "A plaque detailing the historical significance of The Game Of Life in the field of computer science, and that the mural underfoot is a representation of the game in action."
+
+/obj/structure/sign/plaques/tram
+	name = "\improper tram information plate"
+	desc = "A plate showing details from the manufacturer about this SkyyTram Mk III, serial number XC7587TG2022. We are not responsible for any injuries or fatalites caused by usage of the tram. For technical assistance in the event of malfunction please contact: (the info seems to be scratched up to the point of being unreadable.)"
```

Comment:
```suggestion
	desc = "A plate showing details from the manufacturer about this NTram Mk III, serial number XC7587TG2022. We are not responsible for any injuries or fatalites caused by usage of the tram. For technical assistance in the event of malfunction please contact: (the info seems to be scratched up to the point of being unreadable.)"
```
Inserting coder character's into the codebase is generally poor form.
![image](https://user-images.githubusercontent.com/86872535/209525407-1535fd40-361b-490b-bb90-801cebac9617.png)

Reply:
NT-ram

Reply:
given we have pets with full names after coders i dont see an issue with it, since it's also an actual train line name but with an extra y https://en.wikipedia.org/wiki/SkyTrain_(Vancouver)

Reply:
just do N-Tram and avoid all of this mess because people will notice this again in two months and make a big fuss which i don't want

Reply:
can try to think of a better name since that's kinda - weh

Reply:
Changed to a lore friendly company from the code.